<launch>
    <node name="planner_avc" pkg="rr_common" type="planner" output="screen" >
        <!-- 15 deg and 30 deg, respectively -->
        <param name="steer_stddev"     type="double" value="0.261799" />
        <param name="max_steer_angle"  type="double" value="0.523599" />

        <param name="obstacle_cloud_topic"           value="/map"/>
        <param name="path_iterations"  type="int"    value="700" />
        <param name="max_speed"        type="double" value="1.5" />
        <param name="path_stage_time"  type="double" value="2.0" />
        <param name="path_stages"      type="int"    value="2" />
        <param name="time_increment"   type="double" value="0.15" />
        <param name="collision_radius" type="double" value="0.4" />

        <param name="alternate_path_threshold"       value="0.3"/>
        <param name="connected_path_distance"        value="0.09"/>
        <param name="min_cluster_pts"                value="8"/>
    </node>
</launch>
